src/design-system/Input.tsx

```diff
@@ -35,3 +36,21 @@ export function useInputValue<T>(props: InputValueProps<T>) {
 }
 
 export type InputBaseProps<T> = InputLabelProps & InputValueProps<T>;
+
+export const StyledInput = styled.input`
+  background: #e0e4e4;
```

Comment:
Can you use a color from Colors.tsx or add it there if it doesn't already exist?